pkg/operation/botanist/component/kubeapiserver/kube_apiserver_test.go

```diff
@@ -56,6 +57,18 @@ import (
 	fakeclient "sigs.k8s.io/controller-runtime/pkg/client/fake"
 )
 
+var (
+	oldGenerateRandomString func(int) (string, error)
+
+	_ = BeforeSuite(func() {
+		oldGenerateRandomString = secretutils.GenerateRandomString
+		secretutils.GenerateRandomString = secretutils.FakeGenerateRandomString
+	})
+	_ = AfterSuite(func() {
+		secretutils.GenerateRandomString = oldGenerateRandomString
+	})
+)
+
```

Comment:
Thanks. I was already tempted to speed this up my PR as well 😄 

You could simplify this into 
```go
var _ = BeforeSuite(func() {
	DeferCleanup(test.WithVar(&secretutils.GenerateRandomString, secretutils.FakeGenerateRandomString))
})
```
(ref https://onsi.github.io/ginkgo/#cleaning-up-our-cleanup-code-defercleanup)